examples/demo.rs

```diff
@@ -400,6 +407,24 @@ fn key_pressed(_app: &App, model: &mut Model, key: Key) {
                 })
                 .collect();
         }
+        Key::S => {
+            // Save model.timeline_data to a JSON file.
+            let json = serde_json::to_string(&model.timeline_data).expect("Serialization of timeline_data failed!");
+            let mut file = File::create("saved_timeline_data.json").expect("Error opening file!");
+            write!(file, "{}", json).expect("Error saving file!");
```

Comment:
nannou has a short-hand function for this: https://docs.rs/nannou/0.13.1/nannou/io/fn.save_to_json.html

So I think this would become:
```rust
nannou::io::save_to_json("saved_timeline_data.json", &model.timeline_data).expect("Error saving file");
```

Reply:
Great, that's so much cleaner! I had to change the path to "./saved_timeline_data.json" for both save and load, otherwise it created a folder called saved_timeline_data and then crashed when it realised it was a folder :grin: Which makes sense on Unix systems and it seems like Path translates the "./" syntax to a valid Windows path too.